### PR TITLE
tools: fix license header check

### PR DESCRIPTION
**Pull request by jonboulle**
Signed-off-by: Jonathan Boulle <jonathanboulle@gmail.com>

### CODE REVIEW

File: .tool/check-license [hunk 1]
```diff
@@ -7,7 +7,10 @@ set -o pipefail
 ret=0
 
 for file in $(find . -type f -iname '*.go' ! -path './vendor/*'); do
-	(head -n3 "${file}" | grep -Eq "(Copyright|generated|GENERATED)") || (echo "${file}:missing license header" && ret=1)
+	if ! head -n3 "${file}" | grep -Eq "(Copyright|generated|GENERATED)"; then
```
Comment (wking) [inline .tool/check-license]: [Trailing whitespace][1].

[1]: https://travis-ci.org/opencontainers/image-spec/builds/169257851#L318

**Comment by vbatts** [comment]
LGTM

**Comment by jonboulle** [comment]
LGTM

**Comment by jonboulle** [comment]
break-fix so self lgtming

**Pull request merged**